### ISSUE TITLE
Uncaught TypeError: Cannot read property 'wrapDateLine' of null 

### ISSUE THREAD

**Issue by clarkewd** [scholarslab/NeatlineFeatures#82]
Omeka Version 2.1.4 with the NeatlineFeatures plugin installed is throwing an error:

```
Uncaught TypeError: Cannot read property 'wrapDateLine' of null 
```

This is coming from the `OpenLayers.min.js` file, it is triggered inside the `isValidLonLat` function and it looks to be called from somewhere in the main `neatline.js` function.

There is also a warning:

```
Warning: you have included the Google Maps API multiple times on this page. This may cause unexpected errors. 
```

Which seems to be caused from the NeatlineFeatures plugin loading the maps API via [NeatlineFeaturesPlugin.php on Line 207](https://github.com/scholarslab/NeatlineFeatures/blob/master/NeatlineFeaturesPlugin.php#L207)

I tried updating the OpenLayers.min.js from version `2.12` to `2.13.1 (Stable)` but this did not fix the problem.

I also tried wrapping the code inside the `isValidLonLat` function in a try/catch block but there was another error that followed regarding the `toGeometry` function.

I also thought that the Javascript resources may be getting loaded out of order but I didn't have any luck even trying to load OpenLayers.min.js before the others - the errors persisted. Here are the JS files in the order they are being loaded:

```
<!-- JavaScripts -->
<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
<script type="text/javascript">
//<!--
jQuery.noConflict();    //-->
</script>
<script type="text/javascript" src="http://maps.google.com/maps/api/js?sensor=false"></script>
<script type="text/javascript" src="/omeka/plugins/Neatline/views/shared/javascripts/payloads/neatline-public.js"></script>
<script type="text/javascript" src="/omeka/plugins/Neatline/views/shared/javascripts/bootstrap.js"></script>
<script type="text/javascript" src="/omeka/plugins/NeatlineWaypoints/views/shared/javascripts/payloads/waypoints-public.js"></script>
<script type="text/javascript" src="http://maps.google.com/maps/api/js?v=3.8&amp;sensor=false"></script>
<script type="text/javascript" src="/omeka/plugins/NeatlineFeatures/views/shared/javascripts/libraries/openlayers/OpenLayers.min.js"></script>
<script type="text/javascript" src="/omeka/plugins/NeatlineFeatures/views/shared/javascripts/libraries/tile.stamen.js"></script>
<script type="text/javascript" src="/omeka/plugins/NeatlineFeatures/views/shared/javascripts/neatline-features-2.0.1-min.js"></script>
<script type="text/javascript" src="/omeka/themes/default/javascripts/vendor/modernizr.js"></script>
```

This is on `Omeka Version 2.1.4` on a Linux machine with these plugins installed:

<img src="http://img38.imageshack.us/img38/1395/7m24.png" alt="screen-2014-03-04_07.59.03.png" />

**Comment by erochest** [comment]
Thanks for all the detail on this. I suspect that one of the dependency has had an API change.

I won't get a chance to look at this until early next week, but I should be able to cut a new release then.

**Comment by clarkewd** [comment]
Thanks Eric! It looks like it works better if `OpenLayers.min.js` and the other NeatlineFeatures scripts  get loaded before the other plugin items get loaded, but there are still some errors in the admin side.

This happens on the item edit page:

```
featureswidget.js:233 - Uncaught TypeError: Cannot read property 'element' of undefined
```

The error code in the console looks like this

<img src="http://img811.imageshack.us/img811/4701/m1rp.png" alt="screen-2014-03-05_01.58.37.png" />

Also, when editing an item the map is visible by default, even when the "use map" checkbox is not checked:

<img src="http://img21.imageshack.us/img21/8793/gcaj.png" alt="screen-2014-03-05_02.00.19.png" />

Click on the "use map" checkbox triggers this error:

```
featureswidget.js:302 - Uncaught TypeError: Cannot read property 'getKml' of undefined
```

Hopefully that will help in debugging. Let me know if I can provide any more information.

This plugin is really cool and I hope we can get it to work with Omeka.

**Comment by erochest** [comment]
Well, it has worked before, so I'm sure we'll get it working again. :)

Thanks for the extra information, also. This will be very helpful.

**Comment by clarkewd** [comment]
Hey Eric - hope you are doing well. Any luck with the dependencies for this? Anything I can do to help?

**Comment by erochest** [comment]
Hey, I've had a couple of other things to look at, but I should get to this in the morning. Thanks for you offer of help. Believe me: you'll be the first to know if there is anything :) Thanks, Eric

**Comment by erochest** [comment]
Hi @clarkewd,

Well, it's nice when I have the fix already done. But it does make reproducing the problem harder.

If it's not a problem for you, could you try out what's on the develop branch right now? If that fixes this problem for you, I'll go ahead and make a new release.

If you won't get a chance to do that, just let me know, and I'll go ahead and release it, maybe by tomorrow.

Thanks,
Eric

**Comment by erochest** [comment]
This should be fixed in 2.0.2 (ade0910032d0291f1a80bbd113979fc47ca6b3fc). Please re-open if this doesn't fix it. Thanks,
